docs/source/standard/document.rst

```diff
@@ -62,19 +65,19 @@ Schema
     Direct link to the document or attachment. 
     
 :datePublished:
-    string, :ref:`date`
+    string, :ref:`date`, auto-generated
     
     |ocdsDescription|
     The date on which the document was first published. 
     
 :dateModified:
-    string, :ref:`date`
+    string, :ref:`date`, auto-generated
     
     |ocdsDescription|
     Date that the document was last modified
     
 :language:
-    string
+    string, optional
 
     Possible values are:
 
```

Comment:
documentOf - invalid list of possible values. Please correct 
:relatedItem: - invalid list of references